oni-lda-c

```diff
@@ -1 +1 @@
-Subproject commit 007ae0cda2df947cc64074c3ca99f8155db5bc3c
+Subproject commit 6cb0e03e7b78e45558a10774a8922b1f948ae8cd
```

Comment:
okay, got it

but we need to explore getting around this - we can't have to edit git files everytime a changes comes to a repo used by another